decidim-ephemeral_participation/lib/decidim/ephemeral_participation/engine.rb

```diff
@@ -17,7 +17,7 @@ class Engine < ::Rails::Engine
         Decidim::AuthorizationHandler.include(Decidim::EphemeralParticipation::AuthorizationHandlerOverride)
         Decidim::Admin::ComponentForm.include(Decidim::EphemeralParticipation::ComponentFormOverride)
         Decidim::Admin::PermissionsForm.include(Decidim::EphemeralParticipation::PermissionsFormOverride)
-        Decidim::System::UpdateOrganizationForm.include(Decidim::EphemeralParticipation::UpdateOrganizationFormOverride)
+        Decidim::System::BaseOrganizationForm.include(Decidim::EphemeralParticipation::UpdateOrganizationFormOverride)
```

Comment:
Change the `spec/lib/overrides_spec.rb` spec removing the entry for:  `/app/forms/decidim/system/update_organization_form.rb` and including `/app/forms/decidim/system/base_organization_form.rb`.

Reply:
@entantoencuanto Once you change it. I will approve the PR, as it seems to be working with the latest deployment, isn't it?